lib/forever/cli.js

```diff
@@ -285,11 +286,12 @@ var getOptions = cli.getOptions = function (file) {
 
   if (configs && configs.length) {
     return configs.map(function (conf) {
-      return assignSpawnWith(objectAssign(clone(options), conf));
+      var startupConfig = setDefaults(objectAssign(clone(options), conf));
```

Comment:
not sure why this is correct btw - this gives config values higher priority than cli flags. Surely this should be reversed?